obspy/io/css/core.py

```diff
@@ -7,11 +7,14 @@
 from future.builtins import *  # NOQA
 
 import os
+import sys
```

Comment:
This comes from an old copy of this file and was removed in master. Please remove.